config/pitest-suppressions/pitest-imports-suppressions.xml

```diff
@@ -1,5 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressedMutations>
+  <mutation unstable="false">
+    <sourceFile>ImportOrderCheck.java</sourceFile>
+    <mutatedClass>com.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck</mutatedClass>
+    <mutatedMethod>setGroups</mutatedMethod>
+    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.ArgumentPropagationMutator</mutator>
+    <description>replaced call to java/util/Arrays::copyOf with argument</description>
+    <lineContent>groups = Arrays.copyOf(packageGroups, packageGroups.length);</lineContent>
```

Comment:
please use UnmodifiableCollectionUtil to avoid this.
pitest suppresison can not be extended.

Reply:
done